openmc/geometry.py

```diff
@@ -42,13 +42,8 @@ def root_universe(self):
     @root_universe.setter
     def root_universe(self, root_universe):
         check_type('root universe', root_universe, openmc.Universe)
-        if root_universe.id != 0:
-            msg = 'Unable to add root Universe "{0}" to Geometry since ' \
-                  'it has ID="{1}" instead of ' \
-                  'ID=0'.format(root_universe, root_universe.id)
-            raise ValueError(msg)
-
         self._root_universe = root_universe
+        self.determine_paths()
```

Comment:
If `determine_paths()` is here, is there ever a reason for the user to call it? If not, should it be made a "private" method? Alternatively, I would suggest that we shouldn't call this method by default here since it will only add overhead to the majority of users who aren't (yet) using the districell/mat functionality.